roles/edpm_download_cache/tasks/install.yml

```diff
@@ -22,3 +22,11 @@
   tags:
     - edpm_podman
     - download_cache
+
+- name: Update packages
+  ansible.builtin.include_role:
+    name: osp.edpm.edpm_bootstrap
+    tasks_from: update_package.yml
+  tags:
+    - edpm_bootstrap
+    - download_cache
```

Comment:
this shoudl not have download_cache

Reply:
more contex on this if i rann a deployment limited to the download_cache
tag i would not expect it to do a dnf update of all packages as a side effect.

we likely will document runign download_cache this way to pre stage images and package prior to an update/upgrade. as such we should make sure it only downloads pacakge/images and does not install them.

Reply:
You're right @SeanMooney 

`download_cache` role should only download the packages without installing them, and it's already done [here](https://github.com/openstack-k8s-operators/edpm-ansible/blob/e53be776f54e6ea1f459e8f5e36af89539eeb47a/roles/edpm_download_cache/tasks/packages.yml#L3-L9)

So i guess we can remove this task entirely.

Reply:
the task is correct we jsut need to remove the - download_cache tag form it so that it does not run when you run just that tag.

we need this to do the package update when you run with 

- edpm_bootstrap

Reply:
oh actully no wrong file ya we don't need this at all 

roles/edpm_bootstrap/tasks/update_package.yml does the install